Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -50,37 +142,72 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         finish()
     }
 
-    @objc func showHideAccuracyRadius() {
-        showsAccuracyRing.toggle()
+    private func customizePuckButton() {
```

Comment:
The name of `customizePuckButton` sounds like it will return `UIButton` but it doesn't.
```suggestion
    private func addCustomizePuckButton() {
```